### ISSUE TITLE
AJAX Spider takes too long to crawl Firing Range

### ISSUE THREAD

**Issue by sshniro** [zaproxy/zaproxy#5571]
**Describe the bug**
I'm using ZAP's AJAX Spider to crawl the resources at https://public-firing-range.appspot.com/ . But the spider continuously crawls the same URL for 1000+ times. Is this a normal expected behavior? The request remains the same but in response, the attribute called `X-Cloud-Trace-Context` differs for all the calls.

**To Reproduce**
Visit https://public-firing-range.appspot.com/ in the browser and click attack Attack with AJAX Spider in ZAP.

**Expected behavior**
To crawl the unique URL only once.

**Screenshots**
Request Tab.
![image](https://user-images.githubusercontent.com/13045528/64783193-7737bc80-d567-11e9-9b8d-be87c12bafca.png)

Response Tab.
![image](https://user-images.githubusercontent.com/13045528/64783281-b8c86780-d567-11e9-9fc5-d37eae467168.png)

**Software versions**
 - ZAP: [2.8.0]
 - Add-on: Name and version [e.g. Passive Scan Rules Alpha v19. https://github.com/zaproxy/zaproxy/wiki/FAQAddonVersions]
 - OS: [Ubuntu 18]
 - Java: [e.g. oracle version "1.8.0_201"]
 - Browser: [e.g. firefox]

**Comment by psiinon** [comment]
I've just tried this again with 10 Headless Firefox instances.
It did stop after 3135 URLs found
The standard spider finds 325 URLs.
Worth keeping this open but changing the name?

**Comment by psiinon** [comment]
Closing as we are now focussing on the Client Spider.
If this issue still occurs with the Client Spider then please raise a new issue.
